crates/sui-json-rpc/src/read_api.rs

```diff
@@ -226,11 +235,19 @@ impl SuiRpcModule for ReadApi {
 
 #[async_trait]
 impl RpcFullNodeReadApiServer for FullNodeApi {
-    async fn dev_inspect_transaction(&self, tx_bytes: Base64) -> RpcResult<DevInspectResults> {
+    async fn dev_inspect_transaction(
+        &self,
+        tx_bytes: Base64,
+        epoch: Option<EpochId>,
```

Comment:
It would be nice to test what happens when the Epoch is outside the valid range(i.e, less than 0 or larger than current epoch).